CoffeeChat/Controller/GoalsViewController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  GoalsViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 9/16/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GoalsViewController: UIViewController {
+
+    // MARK: - Private Data vars
+    private weak var delegate: OnboardingPageDelegate?
+    // TODO: change when networking with backend
+    private var goals: [String] = [
+        "Just chatting",
+        "Finding my people",
+        "Meeting someone different",
+        "Learning from mentors",
+        "Guiding mentees",
+        "Not sure yet"
```

Comment:
A `[String:Bool]` dictionary makes more sense here imo

Reply:
LOL sorry about the screenshot!! my computer literally died on me